src/components/Switch/index.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import propTypes from 'prop-types';
+
+import './style.css';
+
+class Switch extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            chosenParamIndex: 0,
+        };
+    }
+    handleClick(which) {
+        this.setState({chosenParamIndex: which })
+    }
+
+    getClass(index) {
+        return "Switch__Param " + (this.state.chosenParamIndex === index ? "Switch_activeParam" : "");
+    }
+
+    render() {
+        const { params, text } = this.props;
+        return (
+            <div className="Switch">
+                {text}
```

Comment:
why not to wrap text in span explicitly?

Reply:
do you mean to hardcode it? it is a universal switch component and I wanted to have it passed from props